Proxy/Configuration/index.ts

```diff
@@ -1,20 +1,14 @@
-import { Request as ConfigurationRequest } from "./Request"
-import { Response as ConfigurationResponse } from "./Response"
+import { Tokenize as ConfigurationTokenize } from "./Tokenize"
 
 export interface Configuration {
 	id: string
```

Comment:
I think we can remove `id` as well. It will be given by the key or property name used.

Reply:
I can check what it is used for at the moment